js/main.js

```diff
@@ -1,13 +1,223 @@
 /* global _ */
-var content = "";
-var error = "";
 var xhr = new XMLHttpRequest();
 
-// construct a quote
+var singularize = function(word) {
+    word = word.trim();
+    if (word.substr(-1) === 's') {
+        return word.slice(0, -1);
+    } else {
+        return word;
+    }
+};
+
+//
+// Things Warren Ellis Calls "My Little X"
+//
+var ellisMyLittleLabel = function(vocabulary) {
```

Comment:
I've seen some people starting to label their anonymous functions, but I don't know how I feel about it yet. `var blah = function blahMethod () {}` this improves debuggability in the browser, but it can't be the same word as the variable, and its extra typing.